core/templates/dev/head/domain/exploration/InteractionObjectFactory.js

```diff
@@ -48,7 +58,10 @@ oppia.factory('InteractionObjectFactory', [
       confirmed_unclassified_answers: this.confirmedUnclassifiedAnswers,
       customization_args: this.customizationArgs,
       default_outcome: this.defaultOutcome,
-      fallbacks: this.fallbacks,
+      fallbacks: this.fallbacks === undefined ? undefined :
```

Comment:
Ditto. Should never be undefined, I think. Or is there a known special case that I'm overlooking?